service/src/main/java/com/commercetools/pspadapter/tenant/TenantConfig.java

```diff
@@ -27,7 +27,9 @@ public TenantConfig(@Nonnull TenantPropertyProvider tenantPropertyProvider, @Non
         this.sphereClientConfig = SphereClientConfig.of(
                 tenantPropertyProvider.getTenantMandatoryNonEmptyProperty(CT_PROJECT_KEY),
                 tenantPropertyProvider.getTenantMandatoryNonEmptyProperty(CT_CLIENT_ID),
-                tenantPropertyProvider.getTenantMandatoryNonEmptyProperty(CT_CLIENT_SECRET));
+                tenantPropertyProvider.getTenantMandatoryNonEmptyProperty(CT_CLIENT_SECRET),
+                tenantPropertyProvider.getTenantProperty(CT_AUTH_URL).orElse("https://auth.europe-west1.gcp.commercetools.com"),
+                tenantPropertyProvider.getTenantProperty(CT_API_URL).orElse("https://api.europe-west1.gcp.commercetools.com"));
```

Comment:
Can you please add the urls to constants

Reply:
i have added the constants to com.commercetools.util.ServiceConstants